vueui/src/components/view/security/guntracker.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div>
+    <h3>Firearm Control System</h3>
+    <template v-if="screen">
+      <vui-button :params="{ lock: 1 }">Unlock Console</vui-button>
+    </template>
+    <template v-else>
+      <vui-button :params="{ lock: 1 }">Lock Console</vui-button>
+      <h3>Detected Firearms</h3>
+      <table>
+        <tr>
+          <th>User</th>
+          <th>Firearm</th>
+          <th/>
+        </tr>
+        <tr v-for="gun in wireless_firing_pins" :key="gun.ref">
+          <td>{{ gun.registered_info }}</td>
+          <td>{{ gun.gun_name }}</td>
+          <td>
+            <vui-button :class="{'button' : 1, 'selectedoption' : gun.disabled_state}" :params="{ togglepin2: gun.ref }">Disabled</vui-button>
```

Comment:
Again, use of newly addeed classes, alsov button class is unnecessery and use more properly named params that do not make me question sanity like `toggle_disable`.

Reply:
Done